Add display neighboring variants to changelog (#129)
broadinstitute/gnomad-blog: content/changelog/2023-09-display-neighboring-variants-for-variant-table-search.md

---

> title: Displaying neighboring variants when seraching the variant table

Reviewer: ```suggestion
title: Displaying neighboring variants when searching the variant table
```

date: "2023-09-18"
order: 1
---

Previously, performing a search in the Variant table would remove all neighboring rows from view, leaving only the exact search results displayed. This often resulted in a loss of context regarding a variant's position in the table and its surrounding variants.

> Now, there is an option to enable "Display neighboring variants" and when this is checked off, users are still able to search for specific variants but while viewing the surrounding variants in the table. As users search, the table automatically jumps to the first search result, which is highlighted. 

Reviewer: ```suggestion
Now, there is an option to enable the display of nearby variants. When "Display neighboring variants" is selected, users are able to search for specific variants and also look at nearby variants in the table. As users search, the table automatically jumps to the first search result, which is highlighted. 
```
if this is an accurate description

---
title: Displaying neighboring variants when searching the variant table
date: "2023-09-18"
order: 1
---

Previously, performing a search in the Variant table would remove all neighboring rows from view, leaving only the exact search results displayed. This often resulted in a loss of context regarding a variant's position in the table and its surrounding variants.

> Now, there is an option to enable the display of nearby variants. When "Display neighboring variants" is selected, users are able to search for specific variants and also look at nearby variants in the table. As users search, the table automatically jumps to the first search result, which is highlighted.

Reviewer: ```suggestion
Now, there is an option to enable the display of nearby variants. When "Display neighboring variants" is selected, users are able to search for specific variants and also view nearby variants in the table. As users search, the table automatically jumps to the first search result, which is highlighted.
```
just reverting back to view (changed my mind, sorry!)
